internal/grpc/interceptors/eventsmiddleware/conversion.go

```diff
@@ -0,0 +1,16 @@
+package eventsmiddleware
+
+import (
+	collaboration "github.com/cs3org/go-cs3apis/cs3/sharing/collaboration/v1beta1"
+	"github.com/cs3org/reva/pkg/events"
+)
+
+// ShareCreated converts response to event
+func ShareCreated(r *collaboration.CreateShareResponse) events.ShareCreated {
+	return events.ShareCreated{
+		Sharer: r.Share.Creator,
+		//Sharee: r.Share.Grantee, // TODO: unmarshaling fails -> find out why
```

Comment:
the micro nats implementation uses `"encoding/json"` not `jsonpb`, also see https://github.com/asim/go-micro/issues/237